test/com/amazon/ion/BinaryByteArrayIteratorSystemProcessingTest.java

```diff
@@ -23,6 +23,11 @@
 {
     private byte[] myBytes;
 
+    @Override
+    protected int expectedLocalNullSlotSymbolId()
+    {
+        return getStreamingMode() == StreamingMode.NEW_STREAMING_INCREMENTAL ? 0 : 10;
```

Comment:
See https://github.com/amzn/ion-java/pull/355/files?file-filters%5B%5D=#r680259531 for discussion of why this is necessary.

Reply:
PR 355 is big enough that GitHub strained to render it. Reproducing the discussion here as a screenshot for easier access:

![image](https://user-images.githubusercontent.com/611616/134407712-3fa0b0c7-799e-4878-8b8e-3662b502bdcd.png)

The method being overridden isn't in this PR, but if it doesn't have a comment explaining what this function does, consider adding one. Future readers won't have the link to the discussion in PR 355.